datastore/gcloud/aio/datastore/value.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any
+from typing import Dict
+
+
+_JSON_KEYS = {
+    type(None): 'nullValue',
+    bool: 'booleanValue',
+    int: 'integerValue',
+    float: 'doubleValue',
+    # TODO timestampValue
+    # TODO keyValue
+    str: 'stringValue',
+    # TODO blobValue
+    # TODO geoPointValue
+    # TODO entityValue
+    # TODO arrayValue
+}
+
+_VALUE_TYPES = {v: k for k, v in _JSON_KEYS.items()}
```

Comment:
This is not strictly necessary but I find that it makes `from_repr()` easier to read